spec/setup.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+const { EOL } = require('os');
+const { exec } = require('child_process');
+
+const envPath = path.join(__dirname, '..', '.env');
+let envData = '';
+
+const setWhitelistedAccount = () => {
+  // Load env
+  const data = fs.readFileSync(envPath, (err) => {
+    if (err) {
+      throw err;
+    }
+  });
+  envData = data.toString();
+
+  // Parse and remove comments
+  let envArray = data.toString().split(EOL);
+  envArray = envArray.filter((line) => {
+    if (line && !line.startsWith('#')) {
+      return line;
+    }
+  });
+
+  // check for whtelisted_account
+  const whitelist = envArray.find((line) =>
+    line.startsWith('WHITELISTED_ACCOUNTS')
+  );
+  const whitelistIndex = envArray.indexOf(whitelist);
+  if (!whitelist.includes('oppia')) {
+    // Replace the current whitelist with new one
+    const newWhitelist = 'WHITELISTED_ACCOUNTS=oppia';
+    envArray.splice(whitelistIndex, 1, newWhitelist);
+  }
+
+  // set client secret
+  const clientSecretIndex = envArray.findIndex((line) =>
+    line.startsWith('CLIENT_SECRET')
+  );
+  const newClientSecret = 'CLIENT_SECRET="{}"';
+  envArray.splice(clientSecretIndex, 1, newClientSecret);
+
+  // Save new env.
+  const newEnv = envArray.join(EOL);
+  console.log('Updating .env');
+  fs.writeFileSync(envPath, newEnv, (err) => {
+    if (err) {
+      throw err;
+    }
+  });
+};
+
+const runTest = () => {
+  const jasminePath = path.join(
+    __dirname,
+    '..',
+    'node_modules',
+    '.bin',
+    'jasmine'
+  );
```

Comment:
Move in single line.

Reply:
Keeping them all on one line is longer than 80 lines and eslint complains.

Reply:
You can split into two lines.

Reply:
Done.